docs/components/generic/_index.md

```diff
@@ -201,6 +208,28 @@ resp, err := myGeneric.DoCommand(ctx, map[string]interface{}{"command": "example
 
 For more information, see the [Go SDK Code](https://github.com/viamrobotics/api/blob/main/component/generic/v1/generic_grpc.pb.go).
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+**Parameters:**
+
+- `command` [(string)](https://cplusplus.com/reference/string/string/): The command to execute.
```

Comment:
Neither the `command` nor the returned value are `strings`, they're actually our internal [`AttributeMap`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/sdk/common/proto_type.hpp#L13) type.